### PR TITLE
feat(ADE-495): Add user filter to index table

### PR DESCRIPTION
**Pull request by SamMurphyDev**

### CODE REVIEW

File: src/main/resources/js/TableFilter.js [hunk 1]
```diff
@@ -192,7 +206,8 @@ const calculateHiddenRowForColumn = (list) => {
     if (optionsList.length > 0) {
       const rowList = [];
 
-      $(value).children(`span.${columnClass.substring(5)}`).each(
+      const classKey = columnClass.substring(5);
```
Comment (shouvSarker) [inline src/main/resources/js/TableFilter.js:209]: Could we have `5` in a const with a name that denotes what it is and why is it 5?

File: src/main/resources/js/TableFilter.js [hunk 2]
```diff
@@ -250,7 +265,7 @@ $(document).ready(() => {
     lists.forEach(({key, listRef}) => {
       let tempList = [];
 
-      $(`span.${key}`).each((index, value) => {
+      $(`span.${key},a.${key}`).each((index, value) => {
```
Comment (shouvSarker) [inline src/main/resources/js/TableFilter.js:268]: What is `a` in this context? Could we have a more elaborative name?

File: src/main/resources/vm/IndexPage.vm [hunk 3]
```diff
@@ -59,7 +59,19 @@
         </div>
       </div>
     </td>
-    <td class="w-8"></td>
+    <td class="w-8">
+      <div id="user-multiselect-container" class="multiselect-container">
+        <div class="tags-input search-box">
+          <input id="search-user" class="searchable" autocomplete="chrome-off" tabindex="0"
```
Comment (shouvSarker) [inline src/main/resources/vm/IndexPage.vm:65]: why are we setting autocomplete to `chrome-off`? Is this behaviour browser specific?

File: src/main/resources/js/TableFilter.js (hunk 2 above)
Comment (SamMurphyDev) [reply src/main/resources/js/TableFilter.js:268]: It's an `<a>` tag. Unable to change that

File: src/main/resources/vm/IndexPage.vm (hunk 3 above)
Comment (SamMurphyDev) [reply src/main/resources/vm/IndexPage.vm:65]: Unfortunately yes, Chrome doesn't listen to `autocomplete="off"`. This is the decision as to why https://bugs.chromium.org/p/chromium/issues/detail?id=468153#c164

File: src/main/resources/js/TableFilter.js (hunk 1 above)
Comment (SamMurphyDev) [reply src/main/resources/js/TableFilter.js]: `substring` now references a string length as it's stripping out a prefix. The appConstants have been changed to a template literal when required to incorporate the prefix.

**Review by shouvSarker — APPROVED** [review]
Looks good!

**Review by robinMcA — APPROVED** [review]

**Pull request merged**